app/code/Magento/Backend/view/adminhtml/templates/widget/form/renderer/fieldset.phtml

```diff
@@ -53,7 +53,7 @@ if ($isField) {
 
     <?php if (!$element->getNoContainer()) : ?>
         <fieldset class="<?=  $block->escapeHtmlAttr($cssClass) ?>" id="<?=  $block->escapeHtmlAttr($id) ?>">
-        <?php if (strlen($element->getBeforeElementHtml())) : ?>
+        <?php if (strlen(is_string($element->getBeforeElementHtml())) ? $element->getBeforeElementHtml() : '') : ?>
```

Comment:
I believe that we can move this logic into `getBeforeElementHtml` method that prevent twice call
```
$beforeHtml = $this->getData('before_element_html');
return is_string($beforeHtml) ? $beforeHtml : '';
```